### PR TITLE
asd

### PR DESCRIPTION
**Pull request by BuffaloCilantro**

### CODE REVIEW

**Comment by samanehsan** [comment]
Hello! Thank you so much for the pull request, but could you please add a more descriptive title?

File: helpers.py [hunk 1]
```diff
@@ -48,6 +48,15 @@ def choose_number_of_tweets():
...
 def choose_hashtag():
-    """ Modify this function to use return the hashtah #sparkhackathon
+    """ Modify this function to use return the hashtag #sparkhackathon
     """
+    hashtag = "#sparkhackathon"
     return hashtag
+
+
+
+
```
Comment (samanehsan) [inline helpers.py:58]: why are there so many spaces?!?!?!?!?!?!

File: helpers.py [hunk 2]
```diff
@@ -48,6 +48,15 @@ def choose_number_of_tweets():
 ###############################################
 
 def choose_hashtag():
-    """ Modify this function to use return the hashtah #sparkhackathon
+    """ Modify this function to use return the hashtag #sparkhackathon
```
Comment (samanehsan) [inline helpers.py:51]: thank you for fixing this typo!!!!!!!? :fire:

File: helpers.py [hunk 3]
```diff
@@ -48,6 +48,15 @@ def choose_number_of_tweets():
...
     """
+    hashtag = "#sparkhackathon"
     return hashtag
+
+
```
Comment (lmfs1) [inline helpers.py:56]: don't hurt feeling!

**Review by samanehsan — APPROVED** [review]

**Pull request merged**